### PR TITLE
Add social media preview

### PR DESCRIPTION
**Pull request by hroncok**
Not sure if SVG will work, but we can change the image later. Ideally, we'd have some nice set of photos we can randomly pick from.

### CODE REVIEW

File: pyvocz/templates/_base.html [hunk 1]
```diff
@@ -10,6 +10,20 @@
...
         {% endblock styles %}
+        <!-- For Facebook and Twitter: -->
+        <meta property="og:type" content="article">
+        <meta property="og:title" content="{{ self.title()|trim }}">
+        {%- if g.lang_code == 'cs' %}
+        <meta property="og:description" content="Pyva jsou setkání příznivců programovacího jazyka Python a spřízněných technologií.">
+        {% else %}
+        <meta property="og:description" content="Pyvo is a meetups for fans of the Python programming language and related technologies.">
+        {% endif -%}
+        <meta property="og:url" content="{{ request.url }}">
```
Comment (honzajavorek) [inline pyvocz/templates/_base.html]: Will this be full (external) URL or just the path?

**Comment by honzajavorek** [comment]
Otherwise looks good (but I'm not pyvo.cz maintainer). To verify FB, you can use https://developers.facebook.com/tools/debug/ once it's up. To verify Twitter, I have no idea.

File: pyvocz/templates/_base.html (hunk 1 above)
Comment (hroncok) [reply pyvocz/templates/_base.html]: Full.

**Comment by hroncok** [comment]
https://cards-dev.twitter.com/validator

**Comment by encukou** [comment]
This looks unlikely to break anything, but I know too little about these things to provide a *proper* review.
Merged & deployed; you can debug with https://pyvo.cz now.

**Pull request merged**